### ISSUE TITLE
Problem with refresh

### ISSUE THREAD

**Issue by beamilon** [IGS/gEAR#187]
I am trying to create displays for the multigene in Hearing (default). After creating a display, if I refresh, I sometime get a red banner on top "Fail. Sorry, something went wrong: Error loading layouts. Please contact us with this message for help." and datasets different than the ones supposed to be in this profile appear.
And the dropdown menu for the profiles shows 0 in the folders
![image](https://user-images.githubusercontent.com/70719873/151818451-03d8397b-b7b7-4876-8f78-5219b96162fb.png)

**Comment by adkinsrs** [comment]
@beamilon 

Did you get this error by searching from the front page, or through a permalink? I cannot seem to replicate this with a search from the front page under the gEAR "curator" user.

**Comment by adkinsrs** [comment]
actually I just got the error by refreshing my page (which you said in your post that I accidentally glossed over).

**Comment by adkinsrs** [comment]
Here is the error I see

```python
Traceback (most recent call last):
  File "/var/www/cgi/get_user_layouts.cgi", line 106, in <module>
    main()
  File "/var/www/cgi/get_user_layouts.cgi", line 65, in main
    result['selected'] = result['shared_layouts'][0].id;
IndexError: list index out of range
```

@jorvis besides the semicolon at the end of line 65, I'm guessing that this should be a try/except block if the user does not have any "shared_layouts", right?

**Comment by jorvis** [comment]
Fixed in commit d194c80.  Updating production momentarily
